src/components/Layout/Col/Col.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable valid-jsdoc */
```

Comment:
@amje maybe we should disable this rule in config?

Reply:
Or not using jsdoc comments when it's not jsdoc)

Reply:
If there is alternatives, pls tell me)

This this comment is developers best friends. Life will not be the same as before if you start using all the benefits of jsdoc on hover. Vscode has build in support of this feature

Reply:
like this

<img width="440" alt="image" src="https://user-images.githubusercontent.com/27916444/223386111-7cadb687-d32c-4361-aa5e-eb4ad9f47673.png">

Reply:
What exactly those eslint errors are?

Reply:
This is deprecated rule https://eslint.org/docs/latest/rules/valid-jsdoc

<img width="828" alt="image" src="https://user-images.githubusercontent.com/27916444/224246583-12ab0b4f-15ad-4f5b-9a6e-9096587ba9f0.png">